src/ORM/DataObjectSchema.php

```diff
@@ -1329,4 +1389,22 @@ protected function checkRelationClass($class, $component, $relationClass, $type)
             );
         }
     }
+
+    /**
+     * Get the mode that determines which indexes to create for default_sort.
+     */
+    private function getSortIndexMode(string $class): string
+    {
+        $mode = Config::inst()->get($class, 'default_sort_index_mode') ?? DataObjectSchema::SORT_INDEX_MODE_NONE;
```

Comment:
Isn't this supposed to default to SORT_INDEX_MODE_COMPOSITE?

Reply:
The "default" you're probably thinking of is what's set for `DataObject.default_sort_index_mode`.
This isn't particular piece of code isn't setting a default value, it's saying "null is none" as per the PHPDoc for the `SORT_INDEX_MODE_NONE` const.